crates/interpreter/src/opcode.rs

```diff
@@ -125,6 +125,21 @@ where
     core::array::from_fn(|i| outer(table[i]))
 }
 
+/// An error indicating that an opcode is invalid.
+#[derive(Debug, PartialEq, Eq)]
+#[cfg(feature = "parse")]
+pub struct OpCodeError(());
```

Comment:
```suggestion
pub struct OpCodeError;
```

nit: should be enough?

Reply:
that makes it non-instantiable from outside